exonum-java-binding-cryptocurrency-demo/Readme.md

```diff
@@ -69,3 +69,9 @@ $ npm start -- --port=6040 --api-root=http://127.0.0.1:6000 --explorer-root=http
 `--explorer-root` is a root URL of public API address of blockchain explorer.
 
 Ready! Find demo at [http://127.0.0.1:6040](http://127.0.0.1:6040).
+
+## See also
+- The [reference documentation](https://exonum.com/doc/get-started/java-binding).
```

Comment:
I'd perhaps remove the definite article here, to leave just Reference Documentation